guide/ops/requirements.md

```diff
@@ -24,8 +24,8 @@ any binaries for custom blueprints/integrations.
 There are three main consumers of disk space:
 
 * **Static files**: these are the Apache Brooklyn distribution with its own
-  dependencies, plus binaries for custom blueprints and integrations added to
-  the `lib` directory. Note that Brooklyn requires that Java is installed which
+  dependencies, plus OSGi bundles for custom blueprints and integrations added to
+  the `data/cache` directory. Note that Brooklyn requires that Java is installed which
```

Comment:
Custom bundles would be added to `deploy` folder, `data/cache` is an internal detail.